fbu-app/RegisterViewController.m

```diff
@@ -0,0 +1,129 @@
+//
+//  RegisterViewController.m
+//  fbu-app
+//
+//  Created by lucjia on 7/16/19.
+//  Copyright © 2019 lucjia. All rights reserved.
+//
+
+#import "RegisterViewController.h"
+#import "Parse/Parse.h"
+
+@interface RegisterViewController ()
+
+@property (strong, nonatomic) UITextField *usernameField;
+@property (strong, nonatomic) UITextField *emailField;
+@property (strong, nonatomic) UITextField *passwordField;
+@property (strong, nonatomic) UIButton *registerButton;
+@property (strong, nonatomic) UILabel *registerLabel;
+
+@end
+
+@implementation RegisterViewController
+
+- (void)viewDidLoad {
```

Comment:
Since your layout is static, keeping all of the code for formatting in this method is fine. 

But the typical pattern is:
1. Initialize your views in `viewDidLoad`.
2. Set their frame in `viewWillLayoutSubviews`.

This way if you have a view that will change in response to some outside variable, you can set that variable and call `[self layoutSubviews]` to handle reorganizing the views separately from the business logic.

See:
https://developer.apple.com/library/archive/featuredarticles/ViewControllerPGforiPhoneOS/DefiningYourSubclass.html#//apple_ref/doc/uid/TP40007457-CH7-SW1